src/providers/ipa/ipa_id.c

```diff
@@ -291,66 +291,67 @@ static int ipa_initgr_get_overrides_step(struct tevent_req *req)
     int ret;
     struct tevent_req *subreq;
     const char *ipa_uuid;
+    const char *dn;
     struct ipa_initgr_get_overrides_state *state = tevent_req_data(req,
                                         struct ipa_initgr_get_overrides_state);
 
-    DEBUG(SSSDBG_TRACE_LIBS,
-          "Processing group %zu/%zu\n", state->group_idx, state->group_count);
+    for (; state->group_idx < state->group_count; state->group_idx++) {
+        dn = ldb_dn_get_linearized(state->groups[state->group_idx]->dn);
 
-    if (state->group_idx >= state->group_count) {
-        return EOK;
-    }
+        DEBUG(SSSDBG_TRACE_LIBS, "Processing group %s (%zu/%zu)\n",
+              dn, state->group_idx, state->group_count);
 
-    ipa_uuid = ldb_msg_find_attr_as_string(state->groups[state->group_idx],
-                                           state->groups_id_attr, NULL);
-    if (ipa_uuid == NULL) {
-        /* This should never happen, the search filter used to get the list
-         * of groups includes "uuid=*"
-         */
-        DEBUG(SSSDBG_OP_FAILURE,
-              "The group %s has no UUID attribute %s, error!\n",
-              ldb_dn_get_linearized(state->groups[state->group_idx]->dn),
-              state->groups_id_attr);
-        return EINVAL;
-    }
+        ipa_uuid = ldb_msg_find_attr_as_string(state->groups[state->group_idx],
+                                               state->groups_id_attr, NULL);
+        if (ipa_uuid == NULL) {
+            /* This should never happen, the search filter used to get the list
+             * of groups includes "uuid=*" */
```

Comment:
Hi,

I guess you can remove this comment, since it is obviously not true for all code paths. Otherwise I'm fine. The test is working as expected, i.e. it fails if the fix is not applied on the IPA server.

bye,
Sumit

Reply:
Removed.

Reply:
Thanks